src/RpcClient/RpcClient.cs

```diff
@@ -340,6 +340,11 @@ public RpcInvokeResult InvokeScript(byte[] script, params UInt160[] scriptHashes
             return RpcInvokeResult.FromJson(RpcSend("invokescript", parameters.ToArray()));
         }
 
+        public RpcUnclaimedGas GetUnclaimedGas(string address)
+        {
+            return RpcUnclaimedGas.FromJson(RpcSend("getunclaimedgas", address));
```

Comment:
Wait for https://github.com/neo-project/neo-modules/issues/261

Reply:
I will fix #261 ASAP.

Reply:
`get_unclaimed_gas`

Reply:
Shargon,we just created another PR that is removing _ on the core now.

Reply:
I see, but I saw this too https://github.com/neo-project/neo-modules/issues/261#issuecomment-650728723 hahahaha We need to unify criterias